### PR TITLE
[kube-prometheus-stack] Allow setting timeoutSeconds in ValidatingWebhook

### PR DESCRIPTION
**Pull request by mdnix**
Signed-off-by: Marco De Luca <marcodl404@gmail.com>

#### What this PR does / why we need it
In some cases, the 10-second timeout is not enough, resulting in a failed helm upgrade.

After checking the validatingwebhook template, I found that there is no way to define the timeout.
In the mutatingwebhook template the value of "prometheusOperator.admissionWebhooks.timeoutSeconds" is considered. I have adjusted the ValidatingWebhook accordingly.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 42.1.0
+version: 42.1.1
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:25]: New feature should raise minor version.

```suggestion
version: 42.2.0
```

File: charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/validatingWebhookConfiguration.yaml [hunk 2]
```diff
@@ -36,6 +36,7 @@ webhooks:
       {{- if and .Values.prometheusOperator.admissionWebhooks.caBundle (not .Values.prometheusOperator.admissionWebhooks.patch.enabled) (not .Values.prometheusOperator.admissionWebhooks.certManager.enabled) }}
       caBundle: {{ .Values.prometheusOperator.admissionWebhooks.caBundle }}
       {{- end }}
+    timeoutSeconds: {{ .Values.prometheusOperator.admissionWebhooks.timeoutSeconds }}
```
Comment (monotek) [inline charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/validatingWebhookConfiguration.yaml:39]: Please add variable also to values.yaml

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/validatingWebhookConfiguration.yaml (hunk 2 above)
Comment (mdnix) [reply charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/validatingWebhookConfiguration.yaml:39]: It's already in there: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L1619

Comment (monotek) [reply charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/validatingWebhookConfiguration.yaml:39]: Oh, sorry. in this case please use the Bugfix version again.

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/validatingWebhookConfiguration.yaml (hunk 2 above)
Comment (mdnix) [reply charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/validatingWebhookConfiguration.yaml:39]: no worries, i have reverted the version to my initial change

**Review by monotek — APPROVED** [review]

**Pull request merged**